packages/node-red-s1seven-api/lib/verify/locales/en-US/verify.html

```diff
@@ -15,6 +15,19 @@ <h3>Inputs</h3>
       choose the environment that you want to use - app.s1seven.dev or
       app.s1seven.com. Defaults to production (app.s1seven.com).
     </dd>
+    <dt class="optional">version <span class="property-type">1</span></dt>
+    <dd>
+      choose the version of the api to use. It can be set in the config node.
+      Defaults to <code>1</code>
+    </dd>
+    <dt class="optional">
+      mode <span class="property-type">test | live</span>
+    </dt>
+    <dd>
+      allows you to choose which choose mode you want to use -
```

Comment:
'allows you to choose which choose mode' did your copilot write too fast ? ;)
Maybe worth mentioning that the mode from the application or access token 
takes precedence over the value selected in the UI.